src/openfga.ne

```diff
@@ -83,6 +84,15 @@ _word           -> ([a-z] | [A-Z] | [0-9] |  "_" |  "-" | "," | "&" | "+" | "/"
     data => data.flat(3).join('').trim()
 %}
 
+_relation_types -> ":" _optional_space "[" _array_of_types "]" _spacing {%
+    data => data[3]
+%}
+
+
+_array_of_types -> ([a-zA-Z0-9_#,\s]):* {%
```

Comment:
are we allowing - in the type name?  Line 107 suggests we allow `-` but we don't here.  Similarly, `$` as the initial character.

Reply:
Ah! You're right. Added those. TY!

Reply:
Can you use `_naming` here?

Reply:
commas?